platform/backend/backend.md

```diff
@@ -54,27 +78,154 @@ We don’t use Windows as OS for development machines. We use latest macOS or Ub
 
 ## Preferred Gems
 
-* [Puma](http://puma.io) as a Ruby web server
-* [Capistrano](http://capistranorb.com) for deployment
-* [Kaminari](https://github.com/amatsuda/kaminari) for pagination
-* [Draper](https://github.com/drapergem/draper) - Decorators/View-Models for Rails Applications
-* [pundit](https://github.com/elabs/pundit) - minimal authorization through OO design and pure Ruby classes
-* [dotenv](https://github.com/bkeepers/dotenv) for setting shell environment variables
-* [exception_notification](https://github.com/smartinez87/exception_notification) for error/exception notifications from server
-* [AppSignal](https://appsignal.com) - better errors notification server with full debug information about each error
-* [swagger-blocks](https://github.com/fotinakis/swagger-blocks) for generating API documentation in [Swagger](http://swagger.io/specification/) format
-* [rack-timeout](https://github.com/heroku/rack-timeout) - add timeouts to rack applications.
-* [rack-cors](https://github.com/cyu/rack-cors) for configuring CORS headers.
-* [lograge](https://github.com/roidrage/lograge) for production logs squshing.
-* [sidekiq](https://github.com/mperham/sidekiq) for background processing.
-
-## Useful Gems
 
+### Authentication
+
+* [api_authentication](https://github.com/MLSDev/api_authentication)
+* [api_session_recovering](https://github.com/MLSDev/api_session_recovering)
+
+### Authorization
+
+* [pundit](https://github.com/elabs/pundit) - Minimal authorization through OO design and pure Ruby classes.
+* [rolify](https://github.com/RolifyCommunity/rolify) - Role management library with resource scoping.
+
+### Active Record
+
+* [AASM](https://github.com/aasm/aasm) - State machines for Ruby classes.
+* [active_record_union](https://github.com/brianhempel/active_record_union) - UNIONs in ActiveRecord! Adds proper union and union_all methods to ActiveRecord::Relation.
+* [PaperTrail](https://github.com/paper-trail-gem/paper_trail) - PaperTrail lets you track changes to your models' data. It's good for auditing or versioning.
+* [paranoia](https://github.com/rubysherpas/paranoia) - ActiveRecord plugin allowing you to hide and restore records without actually deleting them.
```

Comment:
> README:
Notice:
paranoia has some surprising behaviour (like overriding ActiveRecord's delete and destroy) and is not recommended for new projects. See discard's README for more details.
Paranoia will continue to accept bug fixes and support new versions of Rails but isn't accepting new features.

Reply:
@TeTiRoss, Agree 👍